src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -3444,8 +3450,8 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
     override def rawInfo         = fail(NoType)
     override def companionSymbol = fail(NoSymbol)
   }
-  class StubClassSymbol(owner0: Symbol, name0: TypeName, val missingMessage: String) extends ClassSymbol(owner0, owner0.pos, name0) with StubSymbol
-  class StubTermSymbol(owner0: Symbol, name0: TermName, val missingMessage: String) extends TermSymbol(owner0, owner0.pos, name0) with StubSymbol
+  class StubClassSymbol(owner0: Symbol, name0: TypeName, val missingMessage: String, override val pos: Position) extends ClassSymbol(owner0, owner0.pos, name0) with StubSymbol
+  class StubTermSymbol(owner0: Symbol, name0: TermName, val missingMessage: String, override val pos: Position) extends TermSymbol(owner0, owner0.pos, name0) with StubSymbol
```

Comment:
Rather than adding a override for `pos`, we could just call `setPos` in `newStubSymbol`. That seems simpler, and avoids having to worry about performance interactions with JIT by adding overloads.